src/Persistence/Templates/BuiltInTemplates.cs

```diff
@@ -3,15 +3,20 @@
 
 namespace Microsoft.PowerPlatform.PowerApps.Persistence.Templates;
 
-public static class BuiltInTemplates
+public class BuiltInTemplates
 {
-    public const string App = "Appinfo";
-    public const string Host = "HostControl";
-    public const string Screen = "Screen";
-    public const string Component = "Component";
-    public const string CommandComponent = "CommandComponent";
+    public static readonly (string Name, string Id) App = ("Appinfo", "http://microsoft.com/appmagic/appinfo");
```

Comment:
nit: we should evaluate a `TemplateKey` struct with `Name` and `Id`. Tuples are not very manageable.